src/content/en/tools/workbox/reference-docs/v3.0.1/index-all.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html devsite>
+
+<head>
+  <meta name="project_path" value="/web/tools/workbox/_project.yaml" />
+  <meta name="book_path" value="/web/tools/workbox/_book.yaml" />
+  <meta name="gtm_var" data-key="docType" data-value="reference">
+  <title>Index All</title>
+  <link href="jsdoc.css" rel="stylesheet">
+</head>
+
+<body>
+  <div id="jsdoc-body-container">
+    <div id="jsdoc-content">
+      <div id="jsdoc-content-container">
+        <div id="jsdoc-banner" role="banner">
+        </div>
+        <div id="jsdoc-main" role="main">
+          <div class="symbol-detail-labels">
+            <span class="label label-kind"><small>Version</small></span> &nbsp;&nbsp;&nbsp;
+            <span class="label label-version"><small>v3.0.1</small></span>
+          </div>
+          <ul>
+            <li><a href="module-workbox-build.html#.ManifestEntry">module:workbox-build.ManifestEntry</a></li>
```

Comment:
Here's an example of a URL in the updated docset that refers to the now-non-existent `module-workbox-build.html` file.

This particular file is auto-generated, right? It's really weird that JS Doc would spit out HTML that included a URL referencing an HTML file, but not include the HTML file itself.

Hopefully https://github.com/GoogleChrome/workbox/pull/1405 will help?